modules/ROOT/pages/platform/security/encryption.adoc

```diff
@@ -0,0 +1,99 @@
+[[aura-reference-security]]
+= Encryption
```

Comment:
What is the motivation behind two identical files with different names? The comments above apply in this file as well, whichever one you decide to keep.